fuzz/fuzz_targets/cssparser.rs

```diff
@@ -0,0 +1,96 @@
+#![no_main]
+
+use cssparser::*;
+
+const DEBUG: bool = false;
+
+fn parse_and_serialize(input: &str, preserving_comments: bool) -> String {
+    let mut input = ParserInput::new(input);
+    let mut parser = Parser::new(&mut input);
+    let mut serialization = String::new();
+    let result = do_parse_and_serialize(
+        &mut parser,
+        preserving_comments,
+        TokenSerializationType::nothing(),
+        &mut serialization,
+        0,
+    );
+    if result.is_err() {
+        return String::new();
+    }
+    serialization
+}
+
+fn do_parse_and_serialize<'i>(
+    input: &mut Parser<'i, '_>,
+    preserving_comments: bool,
+    mut previous_token_type: TokenSerializationType,
+    serialization: &mut String,
+    indent_level: usize,
+) -> Result<(), ParseError<'i, ()>> {
+    loop {
+        let token = if preserving_comments {
+            input.next_including_whitespace_and_comments()
+        } else {
+            input.next_including_whitespace()
+        };
+        let token = match token {
+            Ok(token) => token,
+            Err(..) => break,
+        };
+        if DEBUG {
+            for _ in 0..indent_level {
+                print!(" ");
+            }
+            println!("{:?}", token);
+        }
+        if token.is_parse_error() {
+            let token = token.clone();
+            return Err(input.new_unexpected_token_error(token))
+        }
+        let token_type = token.serialization_type();
+        if previous_token_type.needs_separator_when_before(token_type) {
+            serialization.push_str("/**/");
```

Comment:
The spec says:
> If the tokenizer preserves comments, the preserved comment should be used; otherwise, an empty comment (/**/) must be inserted.
> https://drafts.csswg.org/css-syntax/#serialization

So we always insert an empty comment here because we don't support `preserving_comments` now. Ya, should be fine.